clang/lib/Driver/Driver.cpp

```diff
@@ -3785,9 +3785,6 @@ class OffloadingActionBuilder final {
     /// Flag to signal if the user requested device code split.
     bool DeviceCodeSplit = false;
 
-    /// Flag to signal if DAE optimization is turned on.
```

Comment:
Is DAE handling elimination related to this change? Not a blocker - just asking :)

Reply:
It's not related - but given the change of the default - the `EnableDAE` check became irrelevant, so I cleaned it up.

Reply:
I see now. Thanks!